website/docs/ref/enums.doc.js

```diff
@@ -0,0 +1,78 @@
+/* @flow */
+/*
+---
+id: enums
+title: Enums
+permalink: /docs/enums.html
+prev: react.html
```

Comment:
thanks, I had the same feeling but I wasn't sure on how to structure it.
I'll put together a single page for "Utility types".